recipes/openldap/all/conanfile.py

```diff
@@ -62,6 +67,7 @@ def yes_no(v):
             return "yes" if v else "no"
 
         tc = AutotoolsToolchain(self)
+        tc.make_args = self._soelim()
```

Comment:
I think that there is no need to create a function to do that. Apart from that, let's append the value instead of restoring everything.

```suggestion
        if is_apple_os(self):
            tc.make_args.append("SOELIM=soelim" if shutil.which("soelim") else "SOELIM=mandoc_soelim")
```